ComponentTextKit/TextKit/CKTextKitContext.h

```diff
@@ -26,7 +26,8 @@
 - (instancetype)initWithAttributedString:(NSAttributedString *)attributedString
                            lineBreakMode:(NSLineBreakMode)lineBreakMode
                     maximumNumberOfLines:(NSUInteger)maximumNumberOfLines
-                         constrainedSize:(CGSize)constrainedSize;
+                         constrainedSize:(CGSize)constrainedSize
+                   layoutManagerSubclass:(Class)layoutManagerSubclass;
```

Comment:
(I only say this since most large text systems heavily use global caches for performance and stable memory consumption)